cli/gen.go

```diff
@@ -101,8 +101,8 @@ func runGenAutocompleteCmd(cmd *cobra.Command, args []string) error {
 
 var genCmd = &cobra.Command{
 	Use:   "gen [command]",
-	Short: "generate manpages and bash completion file",
-	Long:  "Generate manpages and bash completion file.",
+	Short: "generate auxiliary files",
+	Long:  "Generate manpages, example shell configuration, etc.",
```

Comment:
I'd mention test databases as well in the long explanation and list the test databases which are available (right now only startrek).